### ISSUE TITLE
Wrong type definition for input_number initial value

### ISSUE THREAD

**Issue by Steve9F** [keesschollaart81/vscode-home-assistant#795]
After reading 1 & 2 👆, type your issue here and please include the related YAML code like this:

```YAML
input_number:
  daytime_inclination:
    name: Daytime Inclination
    initial: 0
    min: -50
    max: +50
    step: 0.1
    mode: box
```

The following error notification for the initial definition above:

<em>  Initial value when Home Assistant starts.
  
  https://www.home-assistant.io/integrations/input_number#initial
  
  Incorrect type. Expected "boolean".
</em>

Believe vscode-home-assistant/src/language-service/src/schemas/integrations/input_number.ts edit from
   initial?: boolean
should be 
  initial?: number

**Comment by trdischat** [comment]
I think that this was fixed in https://github.com/keesschollaart81/vscode-home-assistant/pull/723, but it looks like this bug fix did not make it into the v1.11.0 release.

**Comment by frenck** [comment]
This was fixed after that release and is pending release.
Closing as fixed in #723
